Week 1/Extra Practice Problems (Week 1)/0496: Next Greater Element I/elijah.cpp

```diff
@@ -0,0 +1,33 @@
+class Solution {
+public:
+    vector<int> nextGreaterElement(vector<int>& nums1, vector<int>& nums2)
```

Comment:
Similar to "Next Greater Element II",  there's a more efficient approach to the problem